pkg/labels/labels.go

```diff
@@ -139,6 +139,9 @@ const (
 
 	// K8sNamespaceLabel is the key that maps to the namespace for a pod.
 	K8sNamespaceLabel = "io.kubernetes.pod.namespace"
+
+	// PodNamespaceMetaLabels is the key that maps to the labels of a namespace.
+	PodNamespaceMetaLabels = "ns-labels"
```

Comment:
This is k8s specific, please prefix with `K8s`

Reply:
Unfortunately we had the `ns-labels` defined. With the current master, when a policy is imported we translated to:
```
    "ingress": [
      {
        "fromEndpoints": [
          {
            "matchLabels": {
              "k8s:ns-labels.name": "default"
            }
          }
        ]
      }
    ],
```

Reply:
`ns-labels` does not come up in our docs so I assume this is not part of the API for now?